src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -34,31 +34,47 @@ message RnaQuantification {
   // Description
   string description = 3;
 
-  // ID of the ReadGroup providing the reads for the analysis.
-  string read_group_id = 4;
+  // ID(s) of the ReadGroup(s) providing the reads for the analysis.
+  repeated string read_group_ids = 4;
 
-  // List of programIds used in the analysis.
-  repeated string program_ids = 5;
+  // Program can be used to track the provenance of how read data was quantified.
+  message Program {
+    // The command line used to run this program.
+    string command_line = 1;
+
+    // The user specified ID of the program.
+    string id = 2;
+
+    // The name of the program.
+    string name = 3;
+
+    // The ID of the program run before this one.
+    string prev_program_id = 4;
+
+    // The version of the program run.
+    string version = 5;
+  }
+  repeated Program programs = 5;
 
   // List of annotations used.
-  repeated string annotation_ids = 6;
+  repeated string feature_set_ids = 6;
 }
 
-// Identifying information for annotated features.
-message QuantificationGroup {
```

Comment:
well, `FeatureGroup` was the original name for this before it was changed to `QuantificationGroup`  We changed it because of a name/conceptual clash with `FeatureGroup` in `sequence_annotation`  I'm happy to change it back as long as we can keep it distinct from `FeatureSet`